dir.props

```diff
@@ -26,6 +26,11 @@
       <Value>True</Value>
     </AssemblyMetadata>
   </ItemGroup>
+  
+  <!-- Temporary adding this property so that Linux builds will target the net45 version of the task library instead of the .Net Core version -->
```

Comment:
I think this is referring to the MSBuild task assembly. Since we're still on Mono, we have to use the task assembly that is compatible with 4.5 for now, until we can use MSBuild on .NET Core, then I would assume we can remove this.